thorlcr/msort/tsorts1.cpp

```diff
@@ -368,11 +368,11 @@ protected: friend class CSortMerge;
 #if defined(_USE_OPENSSL)
                 if (slave.queryTLS())
                 {
-                    unsigned sortTraceLevel = slave.queryTraceLevel();
+                    Owned<ISecureSocket> ssock = secureContextServer->createSecureSocket(socket.getClear());
                     int tlsTraceLevel = SSLogMin;
-                    if (sortTraceLevel)
+                    unsigned sortTraceLevel = slave.queryTraceLevel();
+                    if (sortTraceLevel > DebugMsgThreshold)
```

Comment:
I think this should be > MyVerboseTracingLevle or similar (which is defined as higher than default) for now.